lib/ansible/modules/cloud/amazon/dynamodb_table.py

```diff
@@ -411,9 +462,20 @@ def main():
     except (NoAuthHandlerFound, AnsibleAWSError) as e:
         module.fail_json(msg=str(e))
 
+    if module.params.get('tags'):
+        try:
+            region, ec2_url, aws_connect_kwargs = get_aws_connection_info(module, boto3=True)
+            boto3_dynamodb = boto3_conn(module, conn_type='client', resource='dynamodb', region=region, endpoint=ec2_url, **aws_connect_kwargs)
+            boto3_sts = boto3_conn(module, conn_type='client', resource='sts', region=region, endpoint=ec2_url, **aws_connect_kwargs)
```

Comment:
Rather than just use boto3 for tagging, might just be worth moving to boto3 as a whole, particularly as that avoids the need for the boto3 sts connection (as boto3's create_table returns a TableArn field)